### PR TITLE
Bread edit-add race condition - Issue/fix-1144

### PR DESCRIPTION
**Pull request by chrisboustead**
Address issue #1144

### CODE REVIEW

File: resources/views/tools/database/edit-add-bread.blade.php [hunk 1]
```diff
@@ -328,24 +332,56 @@ class: textarea.attr('class')
...
+                            // Field is empty
+                            $textArea.val('');
                         }
+                    });
+
+                    // If multilingual, prepare data manually.
+                            @if ($isModelTranslatable)
+                    var multilingual = $('.side-body').data().multilingual;
+                    if(multilingual) {
+                        multilingual.prepareData();
```
Comment (akazorg) [inline resources/views/tools/database/edit-add-bread.blade.php:374]: Honestly, I would rater see the prepareData method being handled in the multilingual.js, besides, we may need to handle this on other views and we would be repeating code.

Comment (chrisboustead) [reply resources/views/tools/database/edit-add-bread.blade.php:374]: I think it's fine for `prepareData` to be handled in multilingual.js, but I don't think multilingual should be submitting the form.  What do you suggest as an alternative?  Happy to take a different approach

Comment (akazorg) [reply resources/views/tools/database/edit-add-bread.blade.php:374]: we're talking on Slack, something will come up here.
Just writing this to let others know about the progress.

**Comment by akazorg** [comment]
@marktopper, please don't merge this, I've sent PR #1148 fixing this.

@chrisboustead thanks for your help!

**Comment by marktopper** [comment]
Should this then be closed now that #1148 is merged?

**Comment by akazorg** [comment]
Yes @marktopper, you can close this.

**Pull request closed without merging**